src/qt/treectrl.cpp

```diff
@@ -1164,19 +1170,13 @@ void wxTreeCtrl::Toggle(const wxTreeItemId& item)
 void wxTreeCtrl::Unselect()
 {
     QTreeWidgetItem *current = m_qtTreeWidget->currentItem();
-
-    if ( current != NULL )
-        current->setSelected(false);
+    if (current != NULL)
```

Comment:
```suggestion
    if ( current != NULL )
```
actually this line should not have been changed at all